src/state/InitState.ts

```diff
@@ -65,19 +66,41 @@ export class InitState implements IState{
         const configurationData = new ConfigurationData().getSelectedConfigurationData(this.cliOptions.network);
         this.logger.info("Making sure that Docker is started and it's correct version...", this.stateName);
         // Check if docker is running and it's the correct version
-        const isCorrectDockerComposeVersion = await this.dockerService.isCorrectDockerComposeVersion();
```

Comment:
Why is this deleted?

Reply:
Sorry I was that the value of `isCorrectDockerComposeVersion` is never referenced and thought it was safe to remove. Putting it back in.

Reply:
You can just remove the assignment to a variable, it is indeed not needed. Same goes to the line below